how-to/sending/creating-prefilled-transaction-beta.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How To Create a Pre-Filled Transaction Link (beta)
+description: Transactions
+tags:
+  - Beta
+  - Transactions
+priority: 40
+date_published: '2020-08-18'
+date_modified: '2020-08-18'
+---
+
+<Alert>
+
+This feature is currently only found on MyCrypto beta, which you can find at [beta.mycrypto.com](https://beta.mycrypto.com/).
+
+</Alert>
+
+Pre-filled transactions can be created by appending certain parameters into the query section of a URL.
+
```

Comment:
Add this please:
Pre-filled transactions are a helpful function that allow for many things, including sending transaction requests to others! Instead of telling someone "Okay, send **this** amount to **this** address on **this** network with **this** data, etc. etc.," it allows you to take all the criteria for the transaction and package it up into one neat URL that you can send to someone.

If you create a pre-filled MyCrypto transaction URL and send it to someone, they can input the URL into their browser and it'll automatically take them to the MyCrypto send page and fill it out with the specified parameters.